.gitignore

```diff
@@ -1,5 +1,6 @@
 composer.lock
 composer.phar
+.phpunit.result.cache
```

Comment:
We seem to have this twice, can it be removed?

Reply:
done